### ISSUE TITLE
guiset and prefset throw exceptions when their respective files (userChrome.css and user.js) do not exist

### ISSUE THREAD

**Issue by willruggiano** [tridactyl/tridactyl#3435]
Tridactyl version: 1.21.0
Firefox version: Mozilla Firefox 87.0
Operating system: mac

-   Brief description of the problem: guiset and prefset throw errors when their respective files (userChrome.css and user.js) do not exist. Automatic profile detection does not work.

-   Steps to reproduce:
To repro automatic profile detection not working:
Unset the profiledir config setting, if set. Run; `guiset <any setting> <any value>`. Result; `# controller in excmd: Error: can't access property "indexOf", text is undefined`. This error is coming from here; https://github.com/tridactyl/tridactyl/blob/ac2b06109383132c5dfbd8ed52472da5cb681747/src/lib/native.ts#L836

To repro non-existent user.js: same as above, but ensure that profiledir is set to the correct path for the current Firefox profile

To repro non-existent userChrome.css:
Set profiledir to the correct path for the current Firefox profile. Run `guiset <any setting> <any value>`. Result; `# controller in excmd: Error: Failed to write 'undefined' to '$PROFILE_DIR/chrome/userChrome.css.tri.bak'.`

I was able to work around this by simply creating these files in the right place (and manually setting the profiledir config setting).

**Comment by bovine3dom** [comment]
Thanks for the detailed report. It is probably related to https://github.com/tridactyl/native_messenger/issues/45

Which version of native are you using? `:native` to check

**Comment by willruggiano** [comment]
0.2.5

**Comment by bovine3dom** [comment]
If you have the time could you see if you see the same behaviour with an older version of the messenger?

`curl -fsSl https://raw.githubusercontent.com/tridactyl/tridactyl/master/native/install.sh -o /tmp/trinativeinstall.sh && sh /tmp/trinativeinstall.sh 1.21.0` will install `0.1.11`

Thanks!

**Comment by willruggiano** [comment]
That did fix the issue. Also, I don't think that the "Automatic profile detection does not work" is actually a problem. I have the firefox dev edition installed (but had the normal, or maybe it was nightly, version installed before), so it seems like the automatic profile detection logic was picking the "default" profile, not the "dev default" profile. I think I saw in the code that you had commented that the auto logic was a little hacky :D Anyways, that logic still does not work but I have set profiledir in my config so it's not a big deal.
